draft-ietf-quic-version-negotiation.md

```diff
@@ -77,7 +77,7 @@ establishment if needed.
 It is beneficial to avoid additional round trips whenever possible, especially
 given that most incremental versions are broadly similar to the previous
 version. This specification also defines a simple version negotiation mechanism
-which leverages similarities between versions and can negotiate between
+that leverages similarities between versions and can negotiate between
```

Comment:
This is unnecessary. 

http://itre.cis.upenn.edu/~myl/languagelog/archives/001484.html
```suggestion
which leverages similarities between versions and can negotiate between
```